pkgs/by-name/py/pyfa/package.nix

```
@@ -0,0 +1,33 @@
+{
+  lib,
+  appimageTools,
+  fetchurl,
+}:
+let
+  pname = "pyfa";
+  version = "v2.58.3";
```

The version cannot be prefixed with any characters
```suggestion
  version = "2.58.3";
```

Also commit message

---

should be fixed now, I also updated the version, and learned something new about interactive rebases :D 
sorry for taking so long.